### PR TITLE
Feature2- Add and Display List Of Banks

### PR DESCRIPTION
**Pull request by MuthuKumari-P**
- [x] Add Bank Name List
- [x] Display Bank Name List
- [x] Naming Conversion issues fixed

### CODE REVIEW

File: src/main/java/in/muthukumari/servlet/AddBankNameServlet.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+		//System.out.println("AddBankNameServlet");
+		// Step 1: Get Form Values
+		String bankName = request.getParameter("bankName");
+		out.println(bankName);
+		// Step 2: Call Service => add Product
+		boolean isAdded = OperationsOnBankList.addBankList(bankName);
+		// Step 3: Decide to which page we should redirect ?
+		if (isAdded) {
+			
+			String errorMessage = "valid Bank Name";
```
Comment (csys-fresher-batch-2021) [inline src/main/java/in/muthukumari/servlet/AddBankNameServlet.java]: Successfully Added

File: src/main/webapp/bankNameList.jsp [hunk 2]
```diff
@@ -20,7 +21,7 @@
 				</tr>
 			<tbody>
 				<%
-				Set<String> banksName = BankDetail.getBankName();
```
Comment (csys-fresher-batch-2021) [inline src/main/webapp/bankNameList.jsp:23]: replace banks , bankList - "banksName"

**Review by ghost — CHANGES_REQUESTED** [review]
review comments added

File: src/main/java/in/muthukumari/servlet/AddBankNameServlet.java (hunk 1 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/servlet/AddBankNameServlet.java]: I have changed.

File: src/main/webapp/bankNameList.jsp (hunk 2 above)
Comment (MuthuKumari-P) [reply src/main/webapp/bankNameList.jsp:23]: I have changed this naming conversion.

**Pull request closed without merging**